### DISCUSSION TITLE
Take Control Switch question

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#603] [General]
Hi,

Just wanted to know what switch.layzspa_wifi_controller_layzspa_take_control actually does?

Thanks

**Comment by visualapproach** [comment]
It takes control over the pump. Otherwise the display is doing the controlling. 6 wire pump displays are dumb and don't control anything in the first place.

**Reply by Captain0Kangaroo** [reply]
My switch is currently off but everything seems to work fine via MQTT. (6 wire)

Any reason to turn it on?

**Reply by visualapproach** [reply]
it's only for 4-wire pumps. It should not be visible on 6-wire models.

**Reply by Captain0Kangaroo** [reply]
<img width="327" alt="control" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/96144685/b8519751-8354-4273-8a07-06843751e7f5">
Mine is a Maldives 6 wire and is visable.

**Reply by visualapproach** [reply]
Yeah ok I meant in the user interface. Anyways it doesn't do anything in this case
